cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/batch/source/DBSource.java

```diff
@@ -76,18 +80,19 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
                                   "authentication. If not, please remove dbUser and retry.");
     String jdbcPluginId = String.format("%s.%s.%s", "source", dbSourceConfig.jdbcPluginType,
                                         dbSourceConfig.jdbcPluginName);
-    Class<? extends Driver> jdbcDriverClass = pipelineConfigurer.usePluginClass(dbSourceConfig.jdbcPluginType,
+    driverClass = pipelineConfigurer.usePluginClass(dbSourceConfig.jdbcPluginType,
```

Comment:
https://github.com/caskdata/cdap/pull/3038#discussion_r33396842 Based on this discussion, I chose to call the destroy method instead and make the driverClass a field